src/js/chrome/entry.js

```diff
@@ -84,8 +84,6 @@ export function chromeInit() {
 }
 
 export function bootstrap(libjwt, initFunc, getUser) {
-  const searchParams = new URLSearchParams(location.search);
```

Comment:
@karelhala - I removed whole logic here as only ROS specific cookie exists. is it ok?